### DISCUSSION TITLE
Request for @TaiDak and @jkaw

### DISCUSSION THREAD

**Discussion by Goldenfreddy0703** [Goldenfreddy0703/Otaku#2] [Q&A]
Hey @TaiDak @jkaw, sorry for bothering you but would you be interested in contributing for Otaku? We can make a group chat in [Keybase](https://keybase.io/) if you guys are interested in joining. 

@jkaw, i know you been super busy during the summer but you would really be a big help for us sense you worked on Kaito Beta in the past. If you have time, pls join us on [Keybase](https://keybase.io/). 

My Keybase: https://keybase.io/goldenfreddy0703

**Comment by TaiDak** [comment]
Apologies, been busy with personal life. I declined because I'm not really a developer and would contribute very little to the project. I don't have the technical knowledge at the moment.

**Reply by Goldenfreddy0703** [reply]
Lol your already on our team, you can't decline now. 😂
